test/Icicle/Test/Source/Convert.hs

```diff
@@ -5,6 +5,7 @@
 {-# OPTIONS_GHC -fno-warn-missing-signatures #-}
 module Icicle.Test.Source.Convert where
 
+import           Icicle.Test.Arbitrary
```

Comment:
I got a *really* surprising/confusing error message when I mentioned "$checkAllWith" but hadn't imported this. It was something along the lines of
```
GHC staging restriction:
cannot mention checkAllWith in a splice because it is defined in the same module; you should move it out to a separate module
```